update-db.js

```diff
@@ -158,4 +200,32 @@ module.exports = function updateDB (print) {
   childProcess.execSync(del + ' caniuse-lite')
 
   print('caniuse-lite has been successfully updated\n')
+
+  var currentBrowsersList
+  if (!browsersListRetrievalError) {
+    try {
+      currentBrowsersList = getBrowsersList()
+    } catch (e) /* istanbul ignore next */ {
+      browsersListRetrievalError = e
+    }
+  }
+
+  if (browsersListRetrievalError) {
+    print(browsersListRetrievalError.stack)
+    print(
+      '\nProblem with browsers list retrieval. ' +
+      'Target browser changes won\'t be shown.\n'
+    )
+  } else {
+    var targetBrowserChanges = diffBrowsersLists(
+      oldBrowsersList,
+      currentBrowsersList
+    )
+    if (targetBrowserChanges) {
+      print('\nTarget browser changes:\n')
+      print(targetBrowserChanges + '\n')
+    } else {
+      print('\nNo target browser changes\n')
```

Comment:
I think we can explicitly mention that there are no changes. Though user can deduce that there are no changes when there are no errors and `Target browser changes:` part is missing

Reply:
Good idea. It shows that everything is safe.